guillotina/db/storages/pg.py

```diff
@@ -568,6 +575,18 @@ def lock(self):
 
         await self.initialize_tid_statements()
 
+    def _get_index_name(self, table_name):
+        if '.' in table_name:
+            _, table_name = table_name.split(".", 1)
+
+        if table_name in ("objects", "blobs"):
+            table_name = table_name[:-1]  # singluar index name
+
+        if self._db_schema is None:
+            return table_name
+        else:
+            return f'{self._db_schema}_{table_name}'
```

Comment:
Thouhgt, there's no need to concat tablename to the index. (The index lands on the schema refered by the table)

Reply:
Hmm, in this case I'll remove this code and use previous implementation